.circleci/config.yml

```diff
@@ -50,17 +49,6 @@ jobs:
               RELEASE_TAG=$(echo "$CIRCLE_TAG" | cut -c 6-)
               docker push "docker.io/weaveworks/helm-operator:${RELEASE_TAG}"
             fi
-  e2e-testing:
-    machine: true
```

Comment:
I had totally missed this. I will see if I can make the build work on a machine as well